PWGCF/EbyEFluctuations/Tasks/RobustFluctuationObservables.cxx

```diff
@@ -0,0 +1,1965 @@
+    histosK0S.add("hMassK0S_cutsOnDaughters_v0Rad_10_20", "hMassK0S_cutsOnDaughters_v0Rad_10_20", kTH2D, {axisK0SptBins, K0ShortMassAxis});
+
+    histosK0S.add("hMassK0S_cut_TPC_boundaries_for_Dau", "hMassK0S_cut_TPC_boundaries_for_Dau", kTH2D, {axisK0SptBins, K0ShortMassAxis});
+
+    histosK0S.add("posDau_pt", "posDau_pt", kTH1D, {axisPt});
+    histosK0S.add("posDau_phi", "posDau_phi", kTH1D, {axisPhi});
+    histosK0S.add("posDau_eta", "posDau_eta", kTH1D, {axisEta});
+    histosK0S.add("posDau_phi_pT_05_10", "posDau_phi_pT_05_10", kTH1D, {axisPhi});
+    histosK0S.add("posDau_phi_pT_10_20", "posDau_phi_pT_10_20", kTH1D, {axisPhi});
+    histosK0S.add("posDau_phi_pT_20_100", "posDau_phi_pT_20_100", kTH1D, {axisPhi});
+    histosK0S.add("posDau_phi_vs_pt", "posDau_phi_vs_pt", kTH2D, {axisPtBinsForPhiGaps, axisPhi});
+    histosK0S.add("posDau_phi_vs_pt_afterCut", "posDau_phi_vs_pt_afterCut", kTH2D, {axisPtBinsForPhiGaps, axisPhi});
+
+    histosK0S.add("negDau_pt", "negDau_pt", kTH1D, {axisPt});
+    histosK0S.add("negDau_phi", "negDau_phi", kTH1D, {axisPhi});
+    histosK0S.add("negDau_eta", "negDau_eta", kTH1D, {axisEta});
+    histosK0S.add("negDau_phi_pT_05_10", "negDau_phi_pT_05_10", kTH1D, {axisPhi});
+    histosK0S.add("negDau_phi_pT_10_20", "negDau_phi_pT_10_20", kTH1D, {axisPhi});
+    histosK0S.add("negDau_phi_pT_20_100", "negDau_phi_pT_20_100", kTH1D, {axisPhi});
+    histosK0S.add("negDau_phi_vs_pt", "negDau_phi_vs_pt", kTH2D, {axisPtBinsForPhiGaps, axisPhi});
+    histosK0S.add("negDau_phi_vs_pt_afterCut", "negDau_phi_vs_pt_afterCut", kTH2D, {axisPtBinsForPhiGaps, axisPhi});
+  }
+
+  // Defining filters for events (event selection)
+  // Processed events will be already fulfilling the event selection requirements
+  // Filter eventFilter = (o2::aod::evsel::sel8 == true);
+  // Filter posZFilter = (nabs(o2::aod::collision::posZ) < cutzvertex);
+
+  // Filter<aod::Tracks> ptFilter = aod::track::pt > 1;
+
+  // void process(aod::TracksIU const& tracks)
+  // void process(aod::Collision const& collision, aod::Tracks &tracks) {
+  // void process(aod::Collision const& collision, soa::Filtered<aod::Tracks> &tracks) {
+  // void process(aod::Collision const& collision, soa::Join<aod::Tracks, aod::TracksExtra, aod::TracksDCA> const& tracks)
+  // void process(aod::Collision const& collision,   aod::fullTrack const& fullTrack)  // one can use the abbreviation FullTrack, which is a predefined join of all three track tables
+  // A list of predefined joins is available in The Data Model section of these documentation pages).
+
+  // The grouping works with any number of children. In the below example the process function is given three arguments. In this case process is run for each collision with the tracks and V0s belonging to the actual collision.
+  // void process(aod::Collision const& collision, aod::Tracks const& tracks, aod::V0s const& v0s)
```

Comment:
Commenting code (what the code does) is correct but leaving code commented is against our rules